testDokka.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
minor: may be it's better to move script and it's description into `scripts` dir to not pollute root folder? if that's will be fine (I understand, that it could become inconvenient because of paths)